src/VarshaT/A5_P1.java

```diff
@@ -0,0 +1,39 @@
+//Find the largest of three numbers using logical and operator
+
+package VarshaT;
+
+import java.util.Scanner;
+
+public class A5_P1 {
+	int x, y, z;
+	void input()
+	{
+		Scanner sc = new Scanner(System.in);
+		System.out.println("enter the 3 no's to be comapred:");
+		x = sc.nextInt();
+		y = sc.nextInt();
+		z = sc.nextInt();
+		sc.close();
+	}
+
+	void compare(int x, int y, int z) {
+		System.out.println("Max number:");
+		if (x > y && x > z) {
+			System.out.println(x);
```

Comment:
write some statement which is big number like
System.out.print(x+" is biggest number.")

like above.